utils.js

```diff
@@ -33,10 +33,10 @@ function filterOas(files) {
       } catch (e) {}
     }
     if (fn.match(/.(yaml|yml)/)) {
```

Comment:
does anyone uppercase their YAML files? if that's a thing, `fn.toLowerCase()`

Reply:
I dont think that's a thing if you mean do people have `spec.YAML`:

Filename extensions | .yaml, .yml
-- | --

https://en.wikipedia.org/wiki/YAML